### DISCUSSION TITLE
UNPKG CDN

### DISCUSSION THREAD

**Discussion by dragomano** [ryangjchandler/alpine-clipboard#38] [Ideas]
Add your script on https://unpkg.com CDN, please

**Comment by ryangjchandler** [comment]
That's nothing something I need to do on my end, you can access literally any package on the npm repository via unpkg.com.

https://unpkg.com/browse/@ryangjchandler/alpine-clipboard@2.3.0/dist/alpine-clipboard.js

**Reply by dragomano** [reply]
Oh, nice. Thank you! I was just looking at a different address.
